xml/public-cloud-setup.xml

```diff
@@ -0,0 +1,565 @@
+        will not receive any updates unless specifically setup to
+        receive updates via a different route such as a private SMT
+        server. Registration after the initial setup also requires
+        an explcit registration of each node in the cluster.</para>
+        <para>When all information is collected, accept your
+        selections/input with
+        <literal>y</literal> to complete the initial setup.</para>
+      </sect3>
+    </sect2>
+  </sect1>
+  <sect1 xml:id="bootstrapping">
+    <title>Bootstrapping a Cluster</title>
+    <para>To finalize the configuration and bootstrap the cluster,
+    log into Velum using your admininstrative e-mail address and
+    password. This will take you to the
+    <emphasis role="bold">Initial CaaS Platform
+    Configuration</emphasis> page. The field
+    <emphasis role="bold">Internal Dashboard
+    FQDN/IP</emphasis>should be preset with the internal IP address
+    of your administrative host. You may also choose to install
+    Helm or change the configuration of the overlay network.
+    Clicking
+    <emphasis role="bold">Next</emphasis> will take you to the cloud
+    framework specific configuration page.</para>
+    <sect2>
+      <title>Configuration</title>
+      <para>Within a Public Cloud environment, bootstrapping a
+      &amp;productname; cluster is performed in an automated
+      manner. You must simply choose the instance type best suited
+      to your intended workloads, size the cluster, and specify a
+      few configuration details. The instances will be created, and
+      automatically joined to the cluster, skipping the
+      <emphasis role="italic">pending</emphasis> state. At this
+      point you can continue with the standard setup
+      procedure.</para>
+      <para>In &amp;productname; version 2.1, cluster size has a
+      minimum of three nodes and a maximum of 50 nodes.</para>
+      <sect3>
+        <title>Amazon Web Services EC2</title>
+        <para>You may select from one of the predefined instance
+        types, hand selected for general container workloads, or
+        choose
+        <emphasis role="italic">Other types...</emphasis> and enter
+        any
+        <emphasis role="italic">instance type</emphasis>, as
+        defined at
+        <link xlink:href="https://aws.amazon.com/ec2/instance-types/">
+        https://aws.amazon.com/ec2/instance-types/</link></para>
+        <para>Two configuration options are required in EC2:</para>
+        <variablelist>
+          <varlistentry>
+            <term>Subnet ID</term>
+            <listitem>
+              <para>The
+              <emphasis role="italic">subnet</emphasis> within which
+              cluster nodes will be attached to the network, in the
+              form
+              <literal>subnet-xxxxxxxx</literal>.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>Security Group ID</term>
```

Comment:
IIRC when I set up a 2.1 cluster on AWS this cluster SG was created automatically by the wizard, and the sg id was populated automatically in the wizard.

Reply:
The ID of the SG that is pre-populated is the SG within which the admin node instances runs, which is not the recommended setup, but we didn't want to leave it blank either. I'll think about how this could be expressed better.